src/main/java/com/pet/bankservice/security/jwt/JwtUserMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.pet.bankservice.security.jwt;
+
+import com.pet.bankservice.entity.User;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+public final class JwtUserMapper {
+    public static JwtUser create(User user) {
```

Comment:
Should we have two separate classes? Can we have only one `User` class?

Reply:
done